### DISCUSSION TITLE
Land use change emissions

### DISCUSSION THREAD

**Discussion by WesIngwersen** [USEPA/supply-chain-factors#28] [Q&A] [answered]
from [arby1005](https://github.com/arby1005) originally posted within #25 

Are there any ways to quantify the land use change emissions of food products (e.g., beef, coffee, chocolate, etc.) by using spend data?

**Comment by WesIngwersen** [comment, marked as answer]
Land use change emissions are not included in the [National GHG Industry Attribution Model](https://doi.org/10.23719/1517571), which as described in the Methods section of the [report ](https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324)is the source of the GHG data used to build the factors. 

These emissions would need to be incoporated into that model in order to build them into the supply chain factors. They can't be integrated separately into a comparable spend-based approach.

**Reply by arby1005** [reply]
Hi Weslngwersen,

Thank you for the answer! It is clear that land use change is not included in the EEIO data. I have a follow-up question on this. If there are any indirect land use change emission, e.g. land use change for soy production for animal feed in poultry industry, will it be incorporated or not? 

Thank you

**Reply by WesIngwersen** [reply]
Duplicate question - This was answered in #31
